docs/DeveloperGuide.md

```diff
@@ -532,7 +532,13 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 * Telegram handle: Username of a [Telegram](https://telegram.org/) user.
 
 
-* Contact details: Name and email of the contact. Phone number, address, telegram handle, video conferencing meeting link, and tag(s) are optional.
+* Prof: A group of professors who are facilitators of a specified module.
+
+
+* TA: A group of teaching assistants who are facilitators of a specified module. 
```

Comment:
Change to Teaching Assistant (TA): A student tutor employed to teach small group lessons in NUS.